### PR TITLE
add sites and spatialreference admin forms

### PR DESCRIPTION
**Pull request by miguelcleon**
adds sites and spatialreference admin forms. Adds sampling feature and cv site type ajax lookup fields. Changes sampling feature fields to use the ajax lookup field in several forms.

### CODE REVIEW

File: odm2admin/forms.py [hunk 1]
```diff
@@ -138,7 +144,7 @@ class unitsInLine(admin.StackedInline):
     extra = 0
 
 
-class FeatureActionsInline(admin.StackedInline):
+class FeatureActionsInline(admin.StackedInline): # AjaxSelectAdminStackedInline
```
Comment (miguelcleon) [inline odm2admin/forms.py:147]: @lsetiawan  I'm trying to use ajax select in the inline forms but I'm not sure how to do it. Any ideas?

**Comment by miguelcleon** [comment]
@lsetiawan  and @emiliom This PR will require sites to be created for sampling features going forward. I think this will be necessary for wofpy so I thought I would just make it required for all sampling features via an inline form.

**Comment by emiliom** [comment]
> require sites to be created for sampling features going forward. I think this will be necessary for wofpy

Great. This is a good thing not just for wofpy. It's the convention/expectation in ODM2.

**Comment by miguelcleon** [comment]
@emiliom Ok I wrote a little script and generated sites for all of my sampling features on production and test for the LCZO. I'll do the same for the other ODM2 databases I've been working on.

**Pull request merged**